coralproject/talk
# [CORL-2456]: update vulnerabilities in packages

## What does this PR do?

These changes address vulnerabilities in packages. Here is the TODO list:
- [x] critical production vulnerabilities
- [ ] high production vulnerabilities
- [ ] low production vulnerabilities
- [ ] critical non-production vulnerabilities

I also updated `npm` to get better logging of vulnerabilities and recommendations on how to fix the issues.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

You'll need to update your npm version to at least 8.3.0. `npm upgrade npm@8.3.0 -g`

Do an install, run, and build Coral locally and confirm that you see no errors.
 
 Run either `npm audit` for all or `npm audit --production` for just production and see the list of vulnerabilities (should not include what's been completed in the TODO list).
 
## How do we deploy this PR?
